### PR TITLE
[PHPSTAN] Fill empty php doc param types (Document)

### PR DESCRIPTION
**Pull request by blankse**
Fix all phpstan level 2 errors in Pimcore\Model\Document namespace with following schema:
```
PHPDoc tag @param has invalid value ($xyz)
```
Before:
```
 [ERROR] Found 2731 errors
```
After:
```
 [ERROR] Found 2632 errors
```

### CODE REVIEW

File: models/Document/PageSnippet.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ protected function update($params = [])
     /**
      * @param bool $setModificationDate
      * @param bool $saveOnlyVersion
-     * @param $versionNote string version note
+     * @param string $versionNote
```
Comment (weisswurstkanone) [inline models/Document/PageSnippet.php]: "version note" description missing

Comment (brusch) [reply models/Document/PageSnippet.php]: I would say the variable name speaks for itself, so it's ok.

**Comment by brusch** [comment]
Again, thank you so much!

File: models/Document/PageSnippet.php (hunk 1 above)
Comment (blankse) [reply models/Document/PageSnippet.php]: Yes I also thought that the description has no added value

**Pull request merged**